core/jobs_test.py

```diff
@@ -789,9 +789,26 @@ def test_each_one_off_class_is_subclass_of_base_job_manager(self):
         for klass in jobs_registry.ONE_OFF_JOB_MANAGERS:
             self.assertTrue(issubclass(klass, jobs.BaseJobManager))
 
+    def test_is_abstract_method(self):
+        class TestMockAbstractClass(jobs.BaseJobManager):
+            """A sample Abstract Class."""
+            pass
+
+        mock_abstract_base_class = [
+            TestMockAbstractClass
+        ]
+        abstract_list_swap = self.swap(
+            jobs, 'ABSTRACT_BASE_CLASSES', mock_abstract_base_class)
+        assert_raises_regexp_abstract_function = self.assertRaisesRegexp(
+            Exception,
+            'Tried to directly create a job using the abstract base*')
```

Comment:
Inline this directly in the with clause.

Reply:
Just this `assert_raises_regexp_abstract_function` or `abstract_list_swap` should also be made inline?

Reply:
Ya, actually make both of them inline.

Reply:
Done, made both of them inline